.travis.yml

```diff
@@ -10,10 +10,10 @@ script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
     - if (julia -e 'VERSION < v"0.5" && exit(1)'); then
         julia -e 'include(joinpath(JULIA_HOME, Base.DATAROOTDIR, "julia", "build_sysimg.jl")); build_sysimg(force=true)';
-        julia -e 'Pkg.clone(pwd()); Pkg.build("ForwardDiff"); Pkg.test("ForwardDiff"; coverage=true)';
+        julia -e 'Pkg.clone(pwd()); Pkg.build("ForwardDiff"); Pkg.checkout("DiffBase"); Pkg.test("ForwardDiff"; coverage=true)';
```

Comment:
Do you still need to do this? These kind of things should be very temporary, otherwise you're not testing against the same versions of things your users will have installed.